airflow/models/taskinstance.py

```diff
@@ -561,7 +579,7 @@ def command_as_list(
     def generate_command(
         dag_id: str,
         task_id: str,
-        run_id: str = None,
+        run_id: str,
```

Comment:
For other reviewers wondering, this method returns a command list (for e.g. subprocess), and passing in a None would not work, so all existing usages of this function should already be passing in a `run_id`, making this change safe.